src/Microsoft.Data.Sqlite/Utilities/NativeLibraryLoader.cs

```diff
@@ -81,7 +81,17 @@ public static bool IsLoaded(string dllName)
         }
 
         private static bool IsDNX()
-            => PlatformServices.Default?.Application.RuntimeFramework.Identifier == "DNX";
+        {
+            dynamic defaultPlatformServices =
```

Comment:
We may be able to just use `AppDomain.CurrentDomain.SetupInformation.TargetFrameworkName`. I'll investigate.